src/main/scala/io/sdkman/changelogs/JavaMigrations.scala

```diff
@@ -305,4 +305,15 @@ class JavaMigrations {
       .insert()
     Seq(Linux64, MacOSX, Windows).foreach(removeVersion("java", "13.ea.21-open", _))
   }
+
+  @ChangeSet(order = "142", id = "142-add_openjdk_java_13-ea-23", author = "eddumelendez")
+  def migrate142(implicit db: MongoDatabase): Unit = {
+    List(
+      Version("java", "13.ea.23-open", "https://download.java.net/java/early_access/jdk13/23/GPL/openjdk-13-ea+23_linux-x64_bin.tar.gz", Linux64),
```

Comment:
You now need to provide `Some[Vendor]` for all Java versions as additional parameter. In this case, it would be `Some(OpenJDK)`.

Reply:
Could I also please ask you to remove the previous version (`13.ea.23-open`) migration please? I feel that we should start cleaning up obsolete migrations and only keep those currently in production for the Java changelog.

Reply:
> remove the previous version (13.ea.23-open)

I think you meant `13.ea.22-open`